src/scripts.js

```diff
@@ -45,6 +45,8 @@ let customerPassword = document.getElementById('password');
 let submitLogin = document.getElementById('login-form-submit');
```

Comment:
I like that you targeted so many things by id instead of class - that's something I need to work on.